src/main/java/teammates/common/datatransfer/attributes/FeedbackResponseCommentAttributes.java

```diff
@@ -89,6 +101,41 @@ public Long getId() {
         return feedbackResponseCommentId;
     }
 
+    /**
+     * Converts comment text in form of string for csv i.e if it contains image, changes it into link.
+     * @return Comment in form of string
+     */
+    public String convertCommentTextToStringForCsv() {
+        String htmlText = commentText.getValue();
+        StringBuilder comment = new StringBuilder(200);
+        comment.append(Jsoup.parse(htmlText).text());
+        convertImageToLinkInComment(comment, htmlText);
+        return SanitizationHelper.sanitizeForCsv(comment.toString());
+    }
+
+    /**
+     * Converts comment text in form of string for instructor results table.
```

Comment:
Then how do I write Javadoc to differentiate between `convertCommentTextToStringForCsv()` and `convertCommentTextToStringForHtml()` because both of them are converting comment text to string.

Reply:
But for different purposes right?